### DISCUSSION TITLE
How does Obsidian integration work?

### DISCUSSION THREAD

**Discussion by usnmustanger** [deathau/markdownload#273] [Q&A]
I've installed and endabled the Obsidian Advanced URI extension in Obsidian, and Obsidian integration is enabled in Markdownload options, but I can't figure out how to get Markdownload to create a note in Obsidian. Is Markdownload supposed to create a note when I click "Download" in the plug in? Currently, all it does is download a .md file, no note is created in Obsidian.
FWIW, I'm using Markdownload in Opera on MacOS Sonoma.

**Comment by liuh886** [comment]
hello,  if you
- install Advanced URI in Obsidian
- enabled Obsidian integration and set the directory correctly

You may need to **turn the extension off and on again**. Then, right-click on the webpage you are going to see "Send to Obsidian" from the new options. 

The similar issue has been discussed in https://forum.obsidian.md/t/help-needed-with-markdownload-web-clipper/54920/8

@deathau I think it would be helpful if the necessary description was included on the settings page or [Steps to follow](https://github.com/deathau/markdownload?tab=readme-ov-file#obsidian-integration).
